hbase-server/src/main/java/org/apache/hadoop/hbase/master/balancer/LoadBalancerStateStore.java

```diff
@@ -15,60 +15,52 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.hadoop.hbase.zookeeper;
+package org.apache.hadoop.hbase.master.balancer;
 
 import java.io.IOException;
-import org.apache.hadoop.hbase.Abortable;
 import org.apache.hadoop.hbase.exceptions.DeserializationException;
-import org.apache.hadoop.hbase.util.Bytes;
+import org.apache.hadoop.hbase.master.MasterStateStore;
+import org.apache.hadoop.hbase.master.region.MasterRegion;
+import org.apache.hadoop.hbase.zookeeper.ZKWatcher;
 import org.apache.yetus.audience.InterfaceAudience;
 import org.apache.zookeeper.KeeperException;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 import org.apache.hadoop.hbase.shaded.protobuf.ProtobufUtil;
 import org.apache.hadoop.hbase.shaded.protobuf.generated.LoadBalancerProtos;
 
 /**
- * Tracks the load balancer state up in ZK
+ * Store the balancer state.
  */
 @InterfaceAudience.Private
-public class LoadBalancerTracker extends ZKNodeTracker {
-  private static final Logger LOG = LoggerFactory.getLogger(LoadBalancerTracker.class);
+public class LoadBalancerStateStore extends MasterStateStore {
```

Comment:
should this class just use SwitchStateStore instead? implementation looks the same, unless I'm missing a key difference

Reply:
The protobuf message is different.